# mehmetfatihdata

In [1]:
import pandas as pd

# Q1:
Kredi limiti (creditlimit) 100.000'den büyük olan tüm müşterilerin (-customers tablosundan) müşteri numarası, adı ve kredi limit bilgilerini creditlimit alanına göre sıralı bir şekilde gösteriniz.

In [2]:
cevap=pd.read_csv("sql_odev_soru_1.csv", index_col=0)
cevap.head(3)

müsteri_adı  kredi_limiti
müsteri_numarası                                            
141                     Euro+ Shopping Channel      227600.0
124               Mini Gifts Distributors Ltd.      210500.0
298                            Vida Sport, Ltd      141300.0

# Q2:
customers tablosundan customernumber bilgisi 151 olan kaydın her sütununu gösteriniz.

In [5]:
cevap2=pd.read_csv("sql_odev_1_soru_2.csv")
cevap2

customerNumber        customerName contactLastName contactFirstName  \
0             151  Muscle Machine Inc           Young             Jeff   

        phone       addressLine1 addressLine2 city state  postalCode country  \
0  2125557413  4092 Furth Circle    Suite 400  NYC    NY       10022     USA   

   salesRepEmployeeNumber  creditLimit  
0                    1286     138500.0

# Q3:
Posta kodu (postalcode) 10022 olan müşterilerin kredi limit bilgilerini azalan sırada gösteriniz.

In [6]:
cevap3=pd.read_csv("sql_odev_1_soru_3.csv")
cevap3

customerNumber          customerName  creditLimit  postalCode
0             151    Muscle Machine Inc     138500.0       10022
1             131     Land of Toys Inc.     114900.0       10022
2             181       Vitachrome Inc.      76400.0       10022
3             424  Classic Legends Inc.      67500.0       10022
4             456       Microscale Inc.      39800.0       10022

BİTTİ. TEŞEKKÜRLER.